sdk/tables/data-tables/test/public/utils/recordedClient.ts

```diff
@@ -8,7 +8,7 @@ import { AzureNamedKeyCredential, AzureSASCredential } from "@azure/core-auth";
 
 import "./env";
 
-const mockAccountName = "fakestorageaccount";
```

Comment:
signature auth methods are sensitive to changes in the account name. So I decided to bring back the account name to allow us to test with playback using AzureNamedKeyCredential

Reply:
is this going to make it harder for someone who isn't you to update the tests? Or would they just need to change the name to the account they recorded with?

Reply:
I think I found a way to keep it replaced so that this doesn't become a problem. Waiting for CI to confirm